attributes/default.rb

```diff
@@ -535,3 +535,16 @@
 # Audit logs
 default['hops']['nn']['audit_log']                 = "false"
 default['hops']['rm']['audit_log']                 = "false"
+
+default['hops']['cadvisor']['dir']                 = "#{node['hops']['dir']}/cadvisor"
+default['hops']['cadvisor']['download-url']        = "#{node['download_url']}/docker/cadvisor"
+default['hops']['cadvisor']['port']                = "4194"
+
+
+default['hops']['docker']['cgroup']['enabled']                       = "true"
+default['hops']['docker']['cgroup']['memory']['hard-limit-default']  = "6GB"
```

Comment:
Why do you have a _default['hops']['docker']['cgroup']['memory']['hard-limit-default']_ and a _default['hops']['docker']['cgroup']['memory']['hard-limit']_, shouldn't it be enough with one of them?

Reply:
Like only keep default['hops']['docker']['cgroup']['memory']['hard-limit'] and just set it to 6GB.